content/en/providers/check-in/sp/_index.md

```diff
@@ -1151,9 +1151,9 @@ urn:mace:egi.eu:aai.egi.eu:vm_operator@fedcloud.egi.eu
 Based on the authentication method selected by the user, the EGI proxy assigns a
 Level of Assurance (LoA), which is conveyed to the SP through both the
 `eduPersonAssurance` attribute and the Authentication Context Class
-(`AuthnContextClassRef`) of the SAML authentication response. EGI AAI currently
-distinguishes between three LoA levels, similarly to the
-[eID Assurance Framework (eIDAF)](http://eur-lex.europa.eu/legal-content/EN/TXT/?uri=OJ:JOL_2015_235_R_0002).
+(`AuthnContextClassRef`) of the SAML authentication response. EGI AAI uses the EGI
```

Comment:
EGI AAI uses the --> EGI Check-in uses Assurance Profiles which distinguish between three LoA levels, similarly to the...

Reply:
Done